tasks/lib/run.js

```diff
@@ -396,7 +418,6 @@ RunUtil.runUntilStopped = function(grunt, options, cb) {
         options.runCompleted(result, method, inputs);
       }
 
-      grunt.log.writeln();
```

Comment:
This is actually important - without it, we don't get a nice space above the _Would you like to run another method?_ prompt, see below:

![Uploading Screenshot from 2015-07-28 15:54:32.png…]()